### DISCUSSION TITLE
1.5.4 Release of Kade Engine

### DISCUSSION THREAD

**Discussion by Kade-github** [KadeArchive/Kade-Engine#1015] [General]
Read the changelog [here](https://kadedev.github.io/Kade-Engine/changelogs/changelog-1.5.4)

<hr /><em>This discussion was created from the release <a href='https://github.com/KadeDev/Kade-Engine/releases/tag/1.5.4'>1.5.4 Release of Kade Engine</a>.</em>

**Comment by JariRamsvik** [comment]
WHY IS IT STILL THE 7z THING AGAIN

**Reply by arceuss** [reply]
smaller filesize

**Reply by bruno12354454** [reply]
![noteSplashes](https://user-images.githubusercontent.com/87202381/125114247-567f1600-e0af-11eb-8604-a557eda2014c.png)

**Reply by bruno12354454** [reply]
![NOTE_assets](https://user-images.githubusercontent.com/87202381/125114288-68f94f80-e0af-11eb-87f0-06d57bab60f7.png)

**Reply by JariRamsvik** [reply]
![tired](https://user-images.githubusercontent.com/86282174/145256076-d1efd3a3-3b8c-49e5-8a67-8ea4352624af.png)

**Comment by ghost** [comment]
**The enemy strum line was supposed to light up when the enemy "press" a note? If yes, this is not happening to me, even in the KE 1.5.3**

**Reply by yardmith** [reply]
look in options

**Reply by ghost** [reply]
In-game options, or in the source code?

**Reply by TheRealWhitty** [reply]
The ingame options you decorative toaster

**Reply by isakube** [reply]
> The ingame options you decorative toaster

I love this

**Comment by SuperSpinDashGamer39** [comment]
hey just wondering which hx file allows downscroll work

**Reply by yardmith** [reply]
PlayState.hx

**Reply by ChillTeensie** [reply]
PlayState.hx, Note.hx

**Comment by Joll17** [comment]
The notes dissapear so much more than before

**Comment by MadeySlime** [comment]
I'm having troubles loading every song / week that isn't part of the senpai week

Edit: The notes are the reason of the crash

**Reply by ghost** [reply]
I'm having this trouble too, you know how to fix?

**Comment by D1g1talDev** [comment]
**I JUST WANT THE 1.6 UPDATE ITS BEEN 1 MONTH AND WE DONT HAVE THE NEW UPDATE MAYBE IS IT FRIDAY NIGHT FUNKIN MIC'ED UP ENGINE OR SOME WHAT**

**Reply by isakube** [reply]
I shall politely ask you to literally shut. We don't have enough commits for a new update.

**Reply by arceuss** [reply]
be patient??

**Reply by TheALt-creator** [reply]
what are commits? Just asking :)

**Comment by BarrierTrio** [comment]
The offset seems to be broken. Whenever I press left or right, it simply flashes between 0 and 0.1.

**Comment by bruno12354454** [comment]
![noteSplashes](https://user-images.githubusercontent.com/87202381/125174003-e2647100-e187-11eb-82c7-8e3afd235be1.png)
![NOTE_assets](https://user-images.githubusercontent.com/87202381/125174006-e7292500-e187-11eb-9eab-dd6f87f51a0a.png)

**Reply by ASHere2021** [reply]
could u plz stop posting inages

**Comment by GatoWasap** [comment]
i can't download the source code

**Reply by ASHere2021** [reply]
just try 1000000000000000000000000000000 times😠😠😠😠😠😠

**Reply by TheRealWhitty** [reply]
wdym

**Reply by NonCanonBF** [reply]
click Source Code (zip)

**Reply by Thatkidwitha-Phobia** [reply]
it says its "invalid" idk what to do i just wanna run a mod

**Reply by ghost** [reply]
> it says its "invalid" idk what to do i just wanna run a mod

Look in your downloads folder on the browser you use and press "Download Anyway" or whatever its called in the browser u use. It should let u download now.

**Comment by HugeAngelNT** [comment]
When i built the game, instead of fnf showing me the intro and then the title screen, it just sent me right directly to the title screen with no intro at all, wich seems kinda sussy, also the icongrid is still the same even after "replacing" the files in the preload folder, i little help would be appreciated.

**Comment by ghost** [comment]
why the hell is this the best version of kade engine

**Reply by ghost** [reply]
Because its the last version of Kade Engine before the coding layouts started to change.

**Comment by NonCanonBF** [comment]
Why does my game, after all assets are loaded, crashes the game after building?

**Comment by CoolGamerTheSpy** [comment]
I cant turn off ghost taping for some reason. Dose anyone know to fix this?
